MsgSender.py

```diff
@@ -0,0 +1,46 @@
+import sys
+import time
+from selenium import webdriver
+
+TIMEOUT = 3
+WHATSAPP_URL = 'https://web.whatsapp.com/'
+
+
+class MsgSender:
```

Comment:
What is this file?
It looks like it's a copy of src/MessageSender.py

Reply:
Thanks, it was a merge error. 
Fixed